flocker/acceptance/testtools.py

```diff
@@ -157,6 +151,10 @@ def _clean_node(test_case, node):
     # http://doc-dev.clusterhq.com/advanced/cleanup.html#removing-zfs-volumes
     # A tool or flocker-deploy option to purge the state of a node does
     # not yet exist. See https://clusterhq.atlassian.net/browse/FLOC-682
+    containers = _run_SSH(22, 'root', node, ['docker', 'ps', '-aq'], None)
```

Comment:
Stopping the containers, removing them, then deleting ZFS volumes is what happens here, in that order. It's done with slightly different Docker commands because we cannot use pipe over these SSH calls - the underlying mechanism uses `Popen` and it's a potential security risk to injection to enable shell access that way.
There might be a good case for simplifying the docs user process for cleaning a node to deploying empty Flocker configs, I don't think that's in the scope of this issue, which is just to make the automated tests a bit faster, but I'd certainly suggest raising that as an issue. The less people following our docs have to do, the better.